pkgs/development/python-modules/fireworks-ai/default.nix

```diff
@@ -25,29 +36,98 @@
   tqdm,
 }:
 
+let
+  asyncstdlib-fw = buildPythonPackage rec {
```

Comment:
Please move these internal packages to the top-level.

Reply:
All requests done

Reply:
@natsukium please re-review, thanks.